README.md

```diff
@@ -14,7 +14,10 @@ For any changes, please create a feature branch and open a PR for it when you fe
 
 ## Getting Started
 
-This is a Ruby on Rails app that uses Postgres since we'll likely depend on Postgres features around geolocation. Ruby and Rails versions are specified in the project's `.ruby-version` and `Gemfile`.
+This is a Ruby on Rails app that uses Postgres since we'll likely depend on Postgres features around geolocation. 
+This project uses `ruby 2.3.0` and `rails 5.1.1`
+
+Click here to install postress [Postgres installation guide](https://wiki.postgresql.org/wiki/Detailed_installation_guides)
 
```

Comment:
Similarly, I think these instructions may be outside the scope of this README. However, if we keep this change, please correct Postgres spelling. Could be changed to something like:

Postgres installation instructions are [here](https://wiki.postgresql.org/wiki/Detailed_installation_guides).

Here being:

`[here](https://wiki.postgresql.org/wiki/Detailed_installation_guides)`